src/app/containers/ArticleMain/index.jsx

```diff
@@ -21,15 +25,60 @@ const componentsToRender = {
   timestamp,
 };
 
+const getISOStringDate = date => new Date(date).toISOString();
+
 const ArticleMain = ({ articleData }) => {
-  const { content, metadata, promo } = articleData;
+  const {
+    brandName,
+    noBylinesPolicy,
+    defaultImage,
+    articleAuthor,
+  } = useContext(ServiceContext);
+  const content = path(['content'], articleData);
```

Comment:
This clutters this file alot, Can you move it to a new file please can be `src/app/containers/ArticleMain/Metadata/index.jsx` that takes `articleData` and returns `MetadataContainer` and `ArticleMetadata` togeather

Reply:
Can the data wrangling stuff just be done in functions?